File: LeetCode/412-fizz-buzz/ex_412.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        ans = []
        for i in xrange(1, n + 1):
            s = ''
            if i % 3 == 0:  s += 'Fizz'
            if i % 5 == 0:  s += 'Buzz'
            if not s:       s = str(i)
            ans.append(s)
        return ans

File: LeetCode/412-fizz-buzz/ex_412.java
public class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> ans = new ArrayList<String>();
        for (int i = 1; i < n + 1; i++) {
            StringBuilder sb = new StringBuilder();
            if (i % 3 == 0)         sb.append("Fizz");
            if (i % 5 == 0)         sb.append("Buzz");
            if (sb.length() == 0)     sb.append(Integer.toString(i));
            ans.add(sb.toString());
        }
        return ans;
    }
}